packages/core/src/conventional-commits/__tests__/get-commits-since-last-release.spec.ts

```diff
@@ -117,6 +117,10 @@ describe('getOldestCommitSinceLastTag', () => {
       });
     });
 
+    afterEach(() => {
```

Comment:
@ghiscoding I got tests passed by mockReset `execSync` after each test.

Reply:
great thanks, it still shows a conflict though, could you fix it? I modified the same unit tests as you so there's a conflict to resolve, thanks

Reply:
Yes sure, it is now resolved 🙏 .